packages/flutter/lib/fix_data.yaml

```diff
@@ -0,0 +1,134 @@
+# Spec file for "dart fix", see <insert URL>.
+
+version: 1
+transforms:
+  # Changes from https://github.com/flutter/flutter/pull/44189.
+  - title: 'Rename to dependOnInheritedElement'
+    date: 2019-11-22
+    element:
+      uris: [ 'material.dart', 'cupertino.dart', 'widgets.dart' ]
```

Comment:
I can see that coming up with the correct list here will be difficult. It will be easy to muss a lib that exports the symbol...

Reply:
That's a valid concern.

The design tradeoff is that we have less information that we can use to ensure that we're selecting the correct change in the case where a declaration has been removed. Having the URIs doesn't absolutely guarantee that we won't do the wrong thing, but it does make it significantly less likely. Unfortunately, I don't have concrete numbers for how often we'd do the wrong thing (either with or without them), but I am concerned about the effect on user trust if the tool makes incorrect changes.